net/microsocks/Makefile

```diff
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=microsocks
+PKG_VERSION:=1.0.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```

Comment:
No, this is not a good approach, I like the previous one because right now, you are downloading file with name "v1.0.3", which is in the dl folder. :-( So, stick to the previous solution, and squash the commits.

Not sure, what was the intention to change it.

Reply:
Intention was to implement 

> I wonder why not use tagged releases from their GitHub? https://github.com/rofl0r/microsocks